src/vivarium_gates_nutrition_optimization/components/pregnancy.py

```diff
@@ -35,7 +39,34 @@ def columns_created(self):
         ] + self.new_children.columns_created
 
     def setup(self, builder: Builder):
-        super().setup(builder)
+        super(BaseDiseaseState, self).setup(builder)
+
```

Comment:
Add comment of what you're doing

Reply:
we have to copy over a lot of setup from VPH's BaseDiseaseState (where `initializes_simulants` is called) into PregnancyState so that we can get "birth_outcome_probabilities" as a required pipeline in initializes_simulants so that the resource manager knows what to call first.